x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/pivot/SchemaUtil.java

```diff
@@ -265,7 +265,7 @@ private static void getSourceFieldMappings(
         );
     }
 
-    private static Map<String, String> extractFieldMappings(FieldCapabilitiesResponse response) {
+    public static Map<String, String> extractFieldMappings(FieldCapabilitiesResponse response) {
```

Comment:
this is used for latest? In this case we should think about moving SchemaUtil out of the pivot package.

Reply:
> this is used for latest? 

Yes

> In this case we should think about moving SchemaUtil out of the pivot package.

Agree. I need to figure out the best class layout for that. `latest` only needs this small method and there are dependencies `SchemaUtil` -> `pivot` in other methods.

Reply:
I've moved the `extractFieldMappings` method to `common.DocumentConversionUtils` class so that it is accessible from both `pivot` and `latest` packages.
This way there are no more dependencies from `latest` to `pivot`.